setup.py

```diff
@@ -30,14 +41,24 @@ def get_version():
 URL = 'http://nilearn.github.com'
 LICENSE = 'new BSD'
 DOWNLOAD_URL = 'http://nilearn.github.com'
-VERSION = get_version()
+VERSION = _VERSION_GLOBALS['__version__']
+
 
 if __name__ == "__main__":
+    if is_installing():
+        eval('_check_module_dependencies(manual_install_only=True)', _VERSION_GLOBALS)
```

Comment:
I believe you can use `_VERSION_GLOBALS['_check_module_dependencies'](manual_install_only=True)` here